database/sql/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -88,13 +88,25 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
       } yield ()
     }
 
+    def fetchAndUpdateStartableWfs(hogGroup: Option[String]) = {
+      for {
+        workflowStoreEntries <- dataAccess.fetchStartableWfsForHogGroup(
+          (limit.toLong, heartbeatTimestampTimedOut, workflowStateExcluded, hogGroup)
+        ).result
+        _ <- DBIO.sequence(
+          workflowStoreEntries map updateForFetched(cromwellId, heartbeatTimestampTo, workflowStateFrom, workflowStateTo)
```

Comment:
Since there's only one `workflowStateFrom` here, what happens if different workflows for the same hog group started in different states?